fbcrawl/items.py

```diff
@@ -487,6 +487,11 @@ def parse_date2(init_date,loader_context):
                     month = months_abbr[date[0].lower()]
                     year = int(date[2])
                     return datetime(year,month,day).date()
+                #'Yesterday', 'at', '10:56'
+                elif date[0].lower() == 'yesterday' and date[1] == 'at':
+                    day = int(str(datetime.now().date()-timedelta(1)).split(sep='-')[2])
+                    month = int(str(datetime.now().date()-timedelta(1)).split(sep='-')[1])
+                    return datetime(year,month,day).date()
```

Comment:
Why the fuck do you need this? Who scrapes for information that occurred yesterday. Very few people even update their profile that often.